objective.py

```diff
@@ -8,27 +8,21 @@
 
 
 class Objective(BaseObjective):
-    min_benchopt_version = "1.3"
+    min_benchopt_version = "1.5"
     name = "TV2D"
 
-    parameters = {'reg': [0.02],
-                  'delta': [0.9],
-                  'isotropy': ["anisotropic", "isotropic"],
-                  'data_fit': ["lsq", "huber"]}
-
-    def __init__(self, reg=0.02, delta=0.1,
```

Comment:
@wassimmazouz was there a particular reason to remove the init ? this slipped through

Reply:
there is no need for `__init__` to pass the parameters as we do this directly with `benchopt`, so removing it avoids having duplicated code, that you need to keep up to date when adding new parameters.